routes/s3handler.js

```diff
@@ -0,0 +1,239 @@
+import express from 'express'
+import CryptoJS from 'crypto-js'
+import bodyParser from 'body-parser'
+import getObjectURLFromS3 from '../src/utils/s3'
+import faceUtils from '../src/utils/face++'
+
+const clientSecretKey = process.env.AWS_SECRET_ACCESS_KEY
+const expectedMinSize = 0
+const expectedMaxSize = 5000000
+const expectedBucket = 'uploadedphotostomatch'
+const expectedHostname = 'uploadedphotostomatch.s3.amazonaws.com'
```

Comment:
Perhaps set these in your `.env` file(s) as well so they can be modified and hidden from prying eyes? 

Also, I think constants are easier to read in ALL_CAPS to distinguish them from other variables in code.

Reply:
good call! will do